index.html

```diff
@@ -12,17 +12,20 @@
   <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
   <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
   <script src="main.js" type="text/javascript"></script>
+  <script src="https://kit.fontawesome.com/3154677c0f.js" crossorigin="anonymous"></script>
+
 </head>
 
 <body>
   <div class="saved">
     <button class="add">
-      + add new sticker
+      add new sticker
+      <i class="fas fa-plus-circle"></i>
     </button>
   </div>
   <div id="container">
     <div class="original sticker" tabindex="1">
-      <button class="close">x</button>
+      <i class="fas fa-times close"></i>
```

Comment:
Keep the `<button>` element. '<i>' is just for icons. We need to know that this is a button with some action.